### PR TITLE
238 metat update

### PR DESCRIPTION
**Pull request by kaijli**
Updated version of https://github.com/microbiomedata/nmdc-edge/pull/238, has working test runs, starting PR to be merged to prod. Needs testing on jet stream.

### CODE REVIEW

File: data/workflow/WDL/metaG/mbin_nmdc.wdl
Comment (chienchi) [inline data/workflow/WDL/metaG/mbin_nmdc.wdl:1]: This seems merged an old version of MAGs wdl.

File: data/workflow/WDL/metaG/mbin_nmdc_output.wdl
Comment (chienchi) [inline data/workflow/WDL/metaG/mbin_nmdc_output.wdl:1]: This seems merged an old version of MAGs wdl.

File: data/workflow/WDL/metaG/nmdcEDGE_assembly.wdl
Comment (chienchi) [inline data/workflow/WDL/metaG/nmdcEDGE_assembly.wdl:1]: This seems merged an old version of assembly wdl.

**Comment by kaijli** [comment]
went through and checked all the differences that github was poorly merging, should mostly be good now? hopefully i didn't overwrite any of my own changes that  i needed to run this, needs a retest.

**Review by chienchi — APPROVED** [review]

**Pull request merged**